### DISCUSSION TITLE
Is it possible to disable slide navigation when only one slide exists?

### DISCUSSION THREAD

**Discussion by fgordilloz** [igordanchenko/yet-another-react-lightbox#116] [Q&A] [answered]
When used with a single element in slides, the previous and next arrows are still displayed and the movement to the same image is executed.

Can this functionality be disabled?

**Comment by igordanchenko** [comment, marked as answer]
Certainly. This use case is covered in the [documentation](https://yet-another-react-lightbox.com/customization#HideNavigationButtons).

**Comment by fgordilloz** [comment]
I have also investigated that with carousel={{ finite: true }} the slide is disabled although still showing the disabled arrows.

Thanks.

**Reply by SurajanShrestha** [reply]
@fgordilloz Thank you so much 🙏. This solved my issue for disabling carousel sliding when there is only one image.

**Comment by ernest-nowacki** [comment]
With the approach we are still able to use arrows (keyboard) to move between slides even though there is only one. Do we have a way to disable this behaviour together with hiding the UI buttons?

**Reply by igordanchenko** [reply]
The [documentation](https://yet-another-react-lightbox.com/customization#HideNavigationButtons) provides a very specific example for this scenario.

**Reply by ernest-nowacki** [reply]
Absolutely, got blind for a moment, thank you.

**Reply by DBAKANG-GIT** [reply]
This helped me as well! Thanks
